adapters/sonobi/sonobi.go

```diff
@@ -115,19 +134,21 @@ func (a *SonobiAdapter) MakeBids(internalRequest *openrtb2.BidRequest, externalR
 	}
 
 	bidResponse := adapters.NewBidderResponseWithBidsCapacity(5)
+	bidResponse.Currency = "USD" // Sonobi only bids in USD
 
 	for _, sb := range bidResp.SeatBid {
 		for i := range sb.Bid {
-			bidType, err := getMediaTypeForImp(sb.Bid[i].ImpID, internalRequest.Imp)
+			bid := sb.Bid[i]
+			bidType, err := getMediaTypeForImp(bid.ImpID, internalRequest.Imp)
+			//bidType, err := getBidType(bid)
```

Comment:
make this function works

Reply:
We can't use mtype right now because we do not return it in the bid response.

Reply:
Since the mtype function is not being used in the current implementation, please remove it for now. We can always add it back when it becomes necessary.

Reply:
I removed it. Thank you.